Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/Util/DatabaseHelper.swift

```diff
@@ -24,6 +55,564 @@ class DatabaseHelper {
         }
     }
 
+    // GRDB upgradeIfRequired
+    private class func upgradeIfRequired(schemaVersion: inout Int32, dbPool: DatabasePool) {
```

Comment:
We will have a lot of code added in these branches. These are not new codes, just the same queries adapted to GRDB and FMDB.

I did look at maybe having some "magic" that would not require these changes but GRDB and FMDB works differently and return different results, thus making that quite hard. :(

Reply:
While it looks that those changes are needed, could we use this change to create some protocol to abstract the DB operations, and then have FMDB implementation ( the current one) and another GRDB implementation?